### PR TITLE
support for 2019 Chrysler Pacifica non-Hybrid

### PR DESCRIPTION
**Pull request by adhintz**
Add support for 2019 Chrysler Pacifica non-Hybrid thanks to @xeb

Add fingerprint for 2017 Pacifica non-Hybrid thanks to irpilot

I ran selfdrive/test/test_fingerprints.py and it ran fine: 
```
Found 79 individual fingerprints
TEST SUCESSFUL
```

### CODE REVIEW

**Comment by rbiasini** [comment]
@adhintz , thanks! Any chance to share cabana drives for those cars?

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Supported Cars
 | Cadillac<sup>3</sup> | ATS 2018                   | Adaptive Cruise      | Yes     | Yes            | 0mph             | 7mph           | Custom<sup>7</sup>|
 | Chrysler             | Pacifica 2017-18           | Adaptive Cruise      | Yes     | Stock          | 0mph             | 9mph           | FCA               |
 | Chrysler             | Pacifica Hybrid 2017-18    | Adaptive Cruise      | Yes     | Stock          | 0mph             | 9mph           | FCA               |
+| Chrysler             | Pacifica 2019              | Adaptive Cruise      | Yes     | Stock          | 0mph             | 39mph          | FCA               |
```
Comment (rbiasini) [inline README.md]: small thing: since it's alphabetic order, this should go one line up (before the hybrid)

Comment (adhintz) [reply README.md]: sure thing, done

**Comment by arne182** [comment]
CHRYSLER PACIFICA 2018
  136 exact key/value matches
  2 mismatches
  Mismatches: {564: Unsupported: 4, this: 8; 924: Unsupported: 3, this: 8

I see mismatches in the fingerprint. I would suggest to split the 2017 and 2018

**Comment by geohot** [comment]
If this pull request is still relevant, please resubmit it to the master branch. This should allow our development flow to work better, we commit to master, you commit to master, everyone commits to master! Learn more: https://medium.com/@comma_ai/a-2020-theme-externalization-13b33326d8b3

**Pull request closed without merging**